modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/extensionflow/MathExtensionTestCase.java

```diff
@@ -0,0 +1,148 @@
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Testing Math Ceil Extension
+ */
+public class MathExtensionTestCase extends CEPIntegrationTest {
+	private static final Log log = LogFactory.getLog(MathExtensionTestCase.class);
+
+	@BeforeClass(alwaysRun = true) public void init() throws Exception {
+
+		super.init(TestUserMode.SUPER_TENANT_ADMIN);
+		String loggedInSessionCookie = getSessionCookie();
+
+		eventSimulatorAdminServiceClient = configurationUtil
+				.getEventSimulatorAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventStreamManagerAdminServiceClient = configurationUtil
+				.getEventStreamManagerAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventPublisherAdminServiceClient = configurationUtil
+				.getEventPublisherAdminServiceClient(backendURL, loggedInSessionCookie);
+		eventProcessorAdminServiceClient = configurationUtil
+				.getEventProcessorAdminServiceClient(backendURL, loggedInSessionCookie);
+	}
+
+	@Test(groups = {
+			"wso2.cep" }, description = "Testing Math Ceil Siddhi Extension") public void siddhiMathExtensionTestScenario()
+			throws Exception {
+		final int messageCount = 3;
+
+		int startESCount = eventStreamManagerAdminServiceClient.getEventStreamCount();
+		int startEPCount = eventPublisherAdminServiceClient.getActiveEventPublisherCount();
+		int startEXPCount = eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount();
+
+		//Add StreamDefinition
+		String streamDefinitionAsString1 = getJSONArtifactConfiguration("extensionflows/math",
+		                                                                "org.wso2.event.sensor.stream_1.0.0.json");
+		eventStreamManagerAdminServiceClient.addEventStreamAsString(streamDefinitionAsString1);
+		String streamDefinitionAsString2 = getJSONArtifactConfiguration("extensionflows/math",
+		                                                                "org.wso2.event.sensorClassifyCeil.stream_1.0.0.json");
+		eventStreamManagerAdminServiceClient.addEventStreamAsString(streamDefinitionAsString2);
+		Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(),
+		                    startESCount + 2);
+
+		//Add Execution Plan
+		String executionPlanAsString =
+				getExecutionPlanFromFile("extensionflows/math", "ExecutionPlan.siddhiql");
+		eventProcessorAdminServiceClient.addExecutionPlan(executionPlanAsString);
+		Assert.assertEquals(
+				eventProcessorAdminServiceClient.getActiveExecutionPlanConfigurationCount(),
+				startEXPCount + 1);
+
+		//Add WSO2Event publisher
+		String eventPublisherConfig =
+				getXMLArtifactConfiguration("extensionflows/math", "Wso2Publisher.xml");
+		eventPublisherAdminServiceClient.addEventPublisherConfiguration(eventPublisherConfig);
+		Assert.assertEquals(eventPublisherAdminServiceClient.getActiveEventPublisherCount(),
+		                    startEPCount + 1);
+
+		EventDto eventDto = new EventDto();
+		eventDto.setEventStreamId("org.wso2.event.sensor.stream:1.0.0");
```

Comment:
You can use a final static variable  since the streamId is used in multiple places